README.md

```diff
@@ -183,14 +172,7 @@ This will autogenerate all of the test cases for which generators exist.  Use
 `git diff` (or your preferred method) to find out which test files have
 changed.  Some exercises will update because someone updated the description
 or other exercise metadata.  Others will change because the actual test suite
-has changed.  If you find that an exercise's test suite (i.e. the actual
```

Comment:
I will need some clarification on this part. Is what I have removed needed? i.e running the command `bin/generate --update <exercise-slug>`